ingestion/src/metadata/ingestion/source/database/dbt_source.py

```diff
@@ -431,8 +431,8 @@ def update_dbt_test_result(self):
                     if dbt_test_completed_at:
                         dbt_timestamp = datetime.strptime(
                             dbt_test_completed_at, "%Y-%m-%dT%H:%M:%S.%fZ"
-                        )
-                        dbt_timestamp = self.unix_time_millis(dbt_timestamp)
```

Comment:
@TeddyCr we can remove the `unix_time_millis` function definition too if it is not being used